src/cts/src/TritonCTS.cpp

```diff
@@ -151,13 +151,17 @@ void TritonCTS::buildClockTrees()
 {
   for (TreeBuilder* builder : *builders_) {
     builder->setTechChar(*techChar_);
-    builder->run();
+    builder->setDb(db_);
+    builder->setLogger(logger_);
+    builder->initBlockages();
+    builder->run(db_);
```

Comment:
When calling ' run ', do you need to set the `db`? It looks like you already set it with `setDb`.

Reply:
For some reason, db_ is reset to nullptr after initBlockages().